### PR TITLE
refactor: move admin controller to user controller

### LINKED ISSUE #160: [REFACTOR] :recycle: rename `admin.controller.ts` to `user.controller.ts`
### Description

in `admin.controller.ts` there is code related to users so we can rename it to improve code quality and maintain code standards

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by YashJain2409**
### Description

move admin.controller.ts to user.controller.ts

resolves #160

### CODE REVIEW

**Review by aayushchugh — COMMENTED** [review]
@YashJain2409  have you updated import in `admin.routes.ts`?

**Comment by YashJain2409** [comment]
yes

**Comment by aayushchugh** [comment]
@YashJain2409 I can't see `admin.routes.ts` file changed

image

**Comment by YashJain2409** [comment]
is it ok now ?

**Review by aayushchugh — APPROVED** [review]

**Comment by aayushchugh** [comment]
@YashJain2409 please edit the PR title and follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

File: src/routes/admin.routes.ts [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-	getAllUsersHandler,
-	patchMarkUserVerifiedHandler,
-} from "../controllers/admin.controller";
 import { createMarketingEmailHandler } from "../controllers/marketingEmail.controller";
 import validateRequest from "../middleware/validateRequest.middleware";
 import { deleteUserSchema, patchMarkUserVerifiedSchema } from "../schemas/admin.schema";
 import { createMarketingEmailSchema } from "../schemas/marketingEmail.schema";
 import { patchMarkUserAdminSchema } from "../schemas/admin.schema";
-import { patchMarkUserAdminHandler } from "../controllers/user.controller";
+import { patchMarkUserAdminHandler,deleteUserHandler,getAllUsersHandler,patchMarkUserVerifiedHandler } from "../controllers/user.controller";
```
Comment (tharun634) [inline src/routes/admin.routes.ts]: Check formatting

File: src/controllers/user.controller.ts [hunk 2]
```diff
@@ -1,11 +1,75 @@
 import { Request, Response } from "express";
 import { StatusCodes } from "http-status-codes";
-import { PatchMarkUserAdminSchema, PatchMarkUserVerifiedSchema } from "../schemas/admin.schema";
+import { PatchMarkUserAdminSchema, PatchMarkUserVerifiedSchema,DeleteUserSchema } from "../schemas/admin.schema";
 
 import { PatchUserSchema } from "../schemas/user.schema";
-import { findUserByUsernameService, updateUserByIdService } from "../services/user.service";
+import { findUserByUsernameService, updateUserByIdService,deleteUserByIdService,findUsersService } from "../services/user.service";
```
Comment (tharun634) [inline src/controllers/user.controller.ts]: Check formatting

File: src/controllers/user.controller.ts [hunk 3]
```diff
@@ -1,11 +1,75 @@
 import { Request, Response } from "express";
 import { StatusCodes } from "http-status-codes";
-import { PatchMarkUserAdminSchema, PatchMarkUserVerifiedSchema } from "../schemas/admin.schema";
+import { PatchMarkUserAdminSchema, PatchMarkUserVerifiedSchema,DeleteUserSchema } from "../schemas/admin.schema";
```
Comment (tharun634) [inline src/controllers/user.controller.ts]: Check formatting

**Review by tharun634 — CHANGES_REQUESTED** [review]

File: src/routes/admin.routes.ts (hunk 1 above)
Comment (aayushchugh) [reply src/routes/admin.routes.ts:7]: @MultiEmail/maintainers should we setup git husky hook to auto format code on commits?

**Comment by YashJain2409** [comment]
is it correct now ?

**Review by tharun634 — APPROVED** [review]

**Comment by aayushchugh** [comment]
@all-contributors please add @YashJain2409 for code

**Pull request merged**